### PR TITLE
fix(InviteFriendsToCommunityPopup): Invite candidates list made scrollable

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Invite candidates list made scrollable. Additionally, bottom padding is fixed to avoid cutting-off the list
too early. Relevant components are added to the Storybook.

Closes: #8004 
Closes: #7603 

### Affected areas

`Storybook`, `InviteFriendsToCommunityPopup`

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Kazam_screencast_00073.webm](https://user-images.githubusercontent.com/20650004/197735150-5e5b3695-85e7-45d1-be35-103cc7d6e7ae.webm)

### CODE REVIEW

File: ui/imports/shared/views/PickedContacts.qml [hunk 1]
```diff
@@ -65,6 +62,8 @@ Item {
             statusListItemIcon.badge.border.color: Theme.palette.baseColor4
             statusListItemIcon.badge.implicitHeight: 14 // 10 px + 2 px * 2 borders
             statusListItemIcon.badge.implicitWidth: 14 // 10 px + 2 px * 2 borders
+
+            MouseArea { anchors.fill: parent }
```
Comment (caybro) [inline ui/imports/shared/views/PickedContacts.qml:66]: Why is this here?

**Review by caybro — COMMENTED** [review]
Otherwise looks cool

**Review by iurimatias — APPROVED** [review]

File: ui/imports/shared/views/PickedContacts.qml (hunk 1 above)
Comment (micieslak) [reply ui/imports/shared/views/PickedContacts.qml]: Let me not answer that question. Already fixed ;)

**Review by caybro — APPROVED** [review]
